sim/dex-data.ts

```diff
@@ -567,6 +510,7 @@ export class Template extends BasicEffect {
 	 * `template.species === template.baseSpecies + '-' + template.forme`
 	 */
 	forme: string;
+	baseForme?: string;
```

Comment:
one attribute that is missing when I remove `Object.assign` and caused us to fail a test.

Reply:
Note to self: remove this.